components/raftstore/src/store/fsm/peer.rs

```diff
@@ -6294,15 +6298,18 @@ where
         if let Some(ForceLeaderState::ForceLeader { time, .. }) = self.fsm.peer.force_leader {
             // Clean up the force leader state after a timeout, since the PD recovery
             // process may have been aborted for some reasons.
-            if time.saturating_elapsed()
-                > cmp::max(
-                    self.ctx.cfg.peer_stale_state_check_interval.0,
-                    Duration::from_secs(60),
-                )
-            {
+            if time.saturating_elapsed() > self.ctx.cfg.peer_stale_state_check_interval.0 {
                 self.on_exit_force_leader();
             }
         }
+        if let Some(state) = &mut self.fsm.peer.unsafe_recovery_state {
+            // Clean up the unsafe recovery state after a timeout, since the PD recovery
+            // process may have been aborted for some reasons.
+            if state.check_timeout(self.ctx.cfg.peer_stale_state_check_interval.0) {
```

Comment:
Isn't this interval's default value too short?

Reply:
5min by default